### PR TITLE
[FIX] pass current context to create popup

### PR DESCRIPTION
**Pull request by hbrunn**
fixes https://github.com/OCA/commission/pull/78#issuecomment-198102192

### CODE REVIEW

**Comment by ivantodorovich** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

File: web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js [hunk 1]
```diff
@@ -22,10 +22,10 @@ openerp.web_widget_one2many_tags = function(instance)
             result.ext.arrow = {
                 onArrowClick: function(e)
                 {
-                    var context = {},
+                    var context = self.build_context(),
                         key = _.str.sprintf(
                             'default_%s', self.field.relation_field)
-                    context[key] = self.field_manager.datarecord.id;
+                    context.add({[key]: self.field_manager.datarecord.id});
```
Comment (guewen) [inline web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js:28]: Shouldn'it be

``` js
context.add({key: self.field_manager.datarecord.id});
```

Comment (hbrunn) [inline web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js:28]: no, then the key's name would be `'key'`. This is the expression you need to have the _value_ of `key` being the key: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Operators/Object_initializer (new in ES6)

Comment (guewen) [inline web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js:28]: Oh of course. Thanks for the link, learned a new thing today :)

**Comment by guewen** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**